src/totp/base32.cpp

```diff
@@ -1,9 +1,8 @@
 // Base32 implementation
+// Source: https://github.com/google/google-authenticator-libpam/blob/master/src/base32.c
 //
 // Copyright 2010 Google Inc.
 // Author: Markus Gutschke
-// Source: https://github.com/google/google-authenticator-libpam/blob/master/src/base32.c
-// Modifications copyright (C) 2017 KeePassXC team <https://keepassxc.org/>
```

Comment:
@TheZ3ro just a small nitpick, doesn't [apache v2.0 requires a modification notice](https://softwareengineering.stackexchange.com/a/220079)? Most of this function was modified in comparison to the original file